From 8a79f3faec3c404977a5c5dd0bd71a3fea2a6648 Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Sun, 29 Oct 2017 16:54:01 +0100
Subject: [PATCH 013/249] wiki: Commonize peripheral names

* Hygrometer vs Humidity
* Fingerprint reader instead of recognition
* Thermometer instead of Temperature
* ...

Change-Id: Icc1af96e0f7670ea9292adae4c333ceaf232733e
---
 _data/devices/FP2.yml     | 2 +-
 _data/devices/ha3g.yml    | 2 +-
 _data/devices/hiaeuhl.yml | 2 +-
 _data/devices/hiaeul.yml  | 2 +-
 _data/devices/hlte.yml    | 2 +-
 _data/devices/hltetmo.yml | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/_data/devices/FP2.yml b/_data/devices/FP2.yml
index c1bb583..ea9170f 100644
--- a/_data/devices/FP2.yml
+++ b/_data/devices/FP2.yml
@@ -27,7 +27,7 @@ network:
 - {tech: 2G, bands: '850 900 1800 1900' }
 - {tech: 3G, bands: '8(900) 2(1900) 1(2100) ' }
 - {tech: 4G, bands: '3(1800) 7(2600) 20(800)' }
-peripherals: [3-axis Compass, 3D Accelerometer, 3D Gyroscope, Ambient light, Dual SIM, FM radio, GPS, Glonass, Proximity]
+peripherals: [3-axis Compass, 3D Accelerometer, 3D Gyroscope, Ambient light, Dual SIM, FM radio, GPS, GLONASS, Proximity sensor]
 ram: 2 GB
 recovery_boot: With the device powered off, hold Volume Up & Power. Release when boot logo appears.
 release: December 2015
diff --git a/_data/devices/ha3g.yml b/_data/devices/ha3g.yml
index f60b10e..b12016e 100644
--- a/_data/devices/ha3g.yml
+++ b/_data/devices/ha3g.yml
@@ -26,7 +26,7 @@ network:
 - {tech: 2G, bands: '850 900 1800 1900 MHz GSM' }
 - {tech: 3G, bands: '850 900 1900 2100 MHz HSDPA' }
 - {tech: 4G, bands: '' }
-peripherals: [Accelerometer, Gyroscope, Proximity sensor, Compass, Barometer, Temperature, Humidity, NFC, GPS, GLONASS]
+peripherals: [Accelerometer, Gyroscope, Proximity sensor, Compass, Barometer, Thermometer, Hygrometer, NFC, GPS, GLONASS]
 ram: 3 GB
 recovery_boot: Vol Up & Home & Power
 release: 2013 Sept
diff --git a/_data/devices/hiaeuhl.yml b/_data/devices/hiaeuhl.yml
index bc442d5..6c98d44 100644
--- a/_data/devices/hiaeuhl.yml
+++ b/_data/devices/hiaeuhl.yml
@@ -26,7 +26,7 @@ network:
 - {tech: 2G, bands: '850 900 1800 1900 MHz GSM' }
 - {tech: 3G, bands: '850 900 1900 2100 MHz HSDPA' }
 - {tech: 4G, bands: '1(2100) 3(1800) 5(850) 7(2600) 8(900) 20(800) 28(700) 38(2600) 40(2300) 41(2500) MHz LTE' }
-peripherals: [Fingerprint recognition, Accelerometer, Gyroscope, Proximity sensor, Compass, Magnetic sensor, GPS, NFC]
+peripherals: [Fingerprint reader, Accelerometer, Gyroscope, Proximity sensor, Compass, Magnetic sensor, GPS, NFC]
 ram: 2/3 GB
 recovery_boot: With the device powered off, hold Volume Down & Power. When the
   HBOOT text appears, release the buttons. Navigate to recovery using the volume
diff --git a/_data/devices/hiaeul.yml b/_data/devices/hiaeul.yml
index 7feb777..08a0702 100644
--- a/_data/devices/hiaeul.yml
+++ b/_data/devices/hiaeul.yml
@@ -26,7 +26,7 @@ network:
 - {tech: 2G, bands: '850 900 1800 1900 MHz GSM' }
 - {tech: 3G, bands: '850 1700(AWS) 1900 2100 MHz HSDPA' }
 - {tech: 4G, bands: '2(1900) 3(1800) 4(1700) 5(850) 7(2600) 12(700) 13(700) 17(700) 19(850) MHz LTE' }
-peripherals: [Fingerprint recognition, Accelerometer, Gyroscope, Proximity sensor, Compass, Magnetic sensor, GPS, NFC]
+peripherals: [Fingerprint reader, Accelerometer, Gyroscope, Proximity sensor, Compass, Magnetic sensor, GPS, NFC]
 ram: 2/3 GB
 recovery_boot: With the device powered off, hold Volume Down & Power. When the
   HBOOT text appears, release the buttons. Navigate to recovery using the volume
diff --git a/_data/devices/hlte.yml b/_data/devices/hlte.yml
index 354c875..e0f6b2b 100644
--- a/_data/devices/hlte.yml
+++ b/_data/devices/hlte.yml
@@ -26,7 +26,7 @@ network:
 - {tech: 2G, bands: '850 900 1800 1900 MHz GSM' }
 - {tech: 3G, bands: '850 900 1900 2100 MHz HSDPA (N9005), 1900 2100 MHz HSPDA (N900P), 900 1900 MHz CDMA (N900P)' }
 - {tech: 4G, bands: '1(2100) 3(1800) 5(850) 7(2600) 8(900) 20(800) (N9005), 25(1900) (N900P)' }
-peripherals: [Accelerometer, Gyroscope, Proximity sensor, Compass, Barometer, Temperature, Humidity, NFC, GPS, GLONASS]
+peripherals: [Accelerometer, Gyroscope, Proximity sensor, Compass, Barometer, Thermometer, Hygrometer, NFC, GPS, GLONASS]
 ram: 3 GB
 recovery_boot: With the device powered off, hold Volume Up, Home & Power. When the blue text appears, release the buttons.
 release: Sept 2013
diff --git a/_data/devices/hltetmo.yml b/_data/devices/hltetmo.yml
index e5819a2..4913f97 100644
--- a/_data/devices/hltetmo.yml
+++ b/_data/devices/hltetmo.yml
@@ -26,7 +26,7 @@ network:
 - {tech: 2G, bands: '850 900 1800 1900 MHz GSM' }
 - {tech: 3G, bands: '850 900 1900 2100 MHz HSDPA (N900T/V/W8), 800 1900 MHz CDMA (N900V)' }
 - {tech: 4G, bands: '4(1700/2100) 17(700) (N900T), 4(1700/2100) 13(700) (N900V), 2(1900) 4(1700/2100) 17(700) (N900W8)' }
-peripherals: [Accelerometer, Gyroscope, Proximity sensor, Compass, Barometer, Temperature, Humidity, NFC, GPS, GLONASS]
+peripherals: [Accelerometer, Gyroscope, Proximity sensor, Compass, Barometer, Thermometer, Hygrometer, NFC, GPS, GLONASS]
 ram: 3 GB
 recovery_boot: With the device powered off, hold Volume Up, Home & Power. When the blue text appears, release the buttons.
 release: Sept 2013
-- 
2.17.1

